cmd/incus/utils.go

```diff
@@ -839,3 +839,21 @@ func sshSFTPServer(ctx context.Context, sftpConn func() (net.Conn, error), entit
 		}()
 	}
 }
+
+func makeJsonable(data any) any {
```

Comment:
It looks like this was the solution you were looking for. It seems like there might be more to the story though.

Reply:
Yeah, that's what I had to use to get the JSON marshaller to function.
It won't handle all possible cases, but it does work with everything in our API right now.